GeneralsMD/Code/Libraries/Source/WWVegas/WWLib/Point.h

```diff
@@ -139,39 +139,39 @@ class TPoint3D : public TPoint2D<T> {
 		TPoint3D(BASECLASS const & rvalue) : TPoint2D<T>(rvalue), Z(0) {}
 		
 		// Equality comparison operators.
-		bool operator == (TPoint3D<T> const & rvalue) const {return(X==rvalue.X && Y==rvalue.Y && Z==rvalue.Z);}
-		bool operator != (TPoint3D<T> const & rvalue) const {return(X!=rvalue.X || Y!=rvalue.Y || Z!=rvalue.Z);}
+		bool operator == (TPoint3D<T> const & rvalue) const {return(this->X==rvalue.X && this->Y==rvalue.Y && Z==rvalue.Z);}
```

Comment:
Again why is this-> needed here? The `using BASECLASS::X;` statement should make it unneccessary?